03.asynchronous/callback.js

```diff
@@ -0,0 +1,70 @@
+import sqlite3 from "sqlite3";
+import timers from "timers/promises";
+
+const db = new sqlite3.Database(":memory:");
+
+db.run(
+  "CREATE TABLE books (id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE)",
+  (err) => {
+    if (err === null) {
+      db.run("INSERT INTO books(title) VALUES('fjord')", function (err) {
```

Comment:
CREATE TABLE とスタイルが揃っていなさそうです 👀 

```suggestion
      db.run("INSERT INTO books (title) VALUES ('fjord')", function (err) {
```